lib/dialects/postgres/connection-manager.js

```diff
@@ -154,6 +154,9 @@ class ConnectionManager extends AbstractConnectionManager {
       }
 
       return new Promise((resolve, reject) => connection.query(query, (error, result) => error ? reject(error) : resolve(result))).then(result => {
+        if (Array.isArray(result)) {
+          result = result.pop();
```

Comment:
Incrementing the version to v7 would disallow us to make sure we still support v6. I am hesitant to bloat the build matrix further.